src/index.js

```diff
@@ -1,14 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { BrowserRouter } from 'react-router-dom';
 import './index.css';
```

Comment:
- [ ] There is an error that appears on the console that prevents rendering your site and that's happening because you deleted the `index.css` file but you still import it here. So Please fix that.

![image](https://user-images.githubusercontent.com/68796232/178001724-3598e120-b30f-46de-98be-673519428379.png)